docs/articles_en/documentation/openvino-ir-format/operation-sets/operations-specifications/movement/strided-slice-1.rst

```diff
@@ -14,6 +14,69 @@ StridedSlice
 
 **Short description**: *StridedSlice* extracts a strided slice of a tensor.
 
+**Detailed description**: The *StridedSlice* operation extracts a slice from a given tensor based on computed indices from the inputs: begin (inclusive of the element at the given index), end (exclusive of the element at the given index), and stride, for each dimension.
+
+The operation takes inputs with the following properties:
+
+* :math:`input` tensor to slice, with N dimensions.
+* :math:`begin, end, stride` inputs - 1D lists of integers of the same length M. **Stride input cannot contain any zeros.**
+* :math:`begin\_mask, end\_mask, new\_axis\_mask, shrink\_axis\_mask, ellipsis\_mask inputs` - bitmasks, 1D lists of integers (0 or 1). **Each mask can have a unique length**. :math:`ellipsis\_mask` can have up to one occurrence of the value 1.
```

Comment:
Can be length of the begin, end masks bigger than rank of input shape? (if other masks are empty)

Reply:
Yes, clarified!